// tsparticles-demo-configs/umd/s/shapeRoundedRect.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Shape Rounded Rect",
        particles: {
            number: {
                value: 80,
                density: {
                    enable: true,
                },
            },
            color: {
                value: "#ff0000",
                animation: {
                    enable: true,
                    speed: 20,
                    sync: true,
                },
            },
            rotate: {
                value: {
                    min: 0,
                    max: 360,
                },
                direction: "clockwise",
                animation: {
                    enable: true,
                    speed: 5,
                    sync: false,
                },
            },
            links: {
                enable: true,
            },
            shape: {
                type: "rounded-rect",
                options: {
                    "rounded-rect": {
                        radius: { min: 5, max: 15 },
                    },
                },
            },
            opacity: {
                value: 0.5,
            },
            size: {
                value: 30,
            },
            move: {
                enable: true,
                speed: 6,
            },
        },
        interactivity: {
            events: {
                onHover: {
                    enable: true,
                    mode: "repulse",
                },
                onClick: {
                    enable: true,
                    mode: "push",
                },
            },
            modes: {
                grab: {
                    distance: 400,
                    links: {
                        opacity: 1,
                    },
                },
                bubble: {
                    distance: 400,
                    size: 40,
                    duration: 2,
                    opacity: 0.8,
                },
                repulse: {
                    distance: 200,
                },
                push: {
                    quantity: 4,
                },
                remove: {
                    quantity: 2,
                },
            },
        },
        background: {
            color: "#000000",
        },
    };
    exports.default = options;
});
