// tsparticles-demo-configs/umd/d/delayOpacity.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Delay Opacity",
        background: {
            color: {
                value: "#000",
            },
        },
        particles: {
            color: {
                value: "#fff",
            },
            number: {
                value: 100,
            },
            opacity: {
                value: { max: 1, min: 0 },
                animation: {
                    enable: true,
                    speed: 5,
                    sync: true,
                    delay: {
                        min: 1,
                        max: 3,
                    },
                    startValue: "max",
                    destroy: "min",
                },
            },
            size: {
                value: 5,
            },
            move: {
                enable: true,
            },
        },
    };
    exports.default = options;
});
