// tsparticles-demo-configs/umd/d/delayStrokeColor.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Delay Stroke Color",
        background: {
            color: {
                value: "#000",
            },
        },
        particles: {
            stroke: {
                color: {
                    value: "#f00",
                    animation: {
                        enable: true,
                        speed: 50,
                        sync: true,
                        delay: {
                            min: 1,
                            max: 3,
                        },
                    },
                },
                width: 3,
            },
            number: {
                value: 100,
            },
            size: {
                value: 5,
            },
            opacity: {
                value: 1,
            },
            color: {
                value: "transparent",
            },
            move: {
                enable: true,
            },
            shape: {
                type: ["triangle", "circle", "square"],
            },
        },
    };
    exports.default = options;
});
