// tsparticles-demo-configs/umd/d/index.js
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "./dataImages", "./delay", "./delayColor", "./delayOpacity", "./delaySize", "./delayStrokeColor", "./destroy", "./disappearing", "./divEvents"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const dataImages_1 = __importDefault(require("./dataImages"));
    const delay_1 = __importDefault(require("./delay"));
    const delayColor_1 = __importDefault(require("./delayColor"));
    const delayOpacity_1 = __importDefault(require("./delayOpacity"));
    const delaySize_1 = __importDefault(require("./delaySize"));
    const delayStrokeColor_1 = __importDefault(require("./delayStrokeColor"));
    const destroy_1 = __importDefault(require("./destroy"));
    const disappearing_1 = __importDefault(require("./disappearing"));
    const divEvents_1 = __importDefault(require("./divEvents"));
    exports.default = {
        dataImages: dataImages_1.default,
        delay: delay_1.default,
        delayColor: delayColor_1.default,
        delayOpacity: delayOpacity_1.default,
        delaySize: delaySize_1.default,
        delayStrokeColor: delayStrokeColor_1.default,
        destroy: destroy_1.default,
        disappearing: disappearing_1.default,
        divEvents: divEvents_1.default,
    };
});
