// tsparticles-demo-configs/umd/w/warp.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Warp",
        fullScreen: false,
        fpsLimit: 120,
        manualParticles: [
            {
                position: {
                    x: 2,
                    y: 2,
                },
            },
            {
                position: {
                    x: 2,
                    y: 98,
                },
            },
            {
                position: {
                    x: 98,
                    y: 2,
                },
            },
            {
                position: {
                    x: 98,
                    y: 98,
                },
            },
            {
                position: {
                    x: 3,
                    y: 1,
                },
            },
            {
                position: {
                    x: 99,
                    y: 2,
                },
            },
            {
                position: {
                    x: 3,
                    y: 2,
                },
            },
            {
                position: {
                    x: 99,
                    y: 1,
                },
            },
        ],
        particles: {
            number: {
                value: 0,
            },
            color: {
                value: "#ffffff",
            },
            shape: {
                type: "circle",
            },
            opacity: {
                value: 1,
            },
            size: {
                value: 3,
            },
            links: {
                enable: true,
                distance: 150,
                color: "#ffffff",
                warp: true,
                opacity: 1,
                width: 1,
            },
            move: {
                enable: false,
                speed: 2,
                outModes: "out",
                warp: true,
            },
        },
        background: {
            color: "#0d47a1",
        },
    };
    exports.default = options;
});
