// tsparticles-demo-configs/umd/c/cards.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Cards",
        particles: {
            number: {
                value: 80,
                density: {
                    enable: true,
                },
            },
            reduceDuplicates: true,
            shape: {
                type: ["spades", "hearts", "diamonds", "clubs"],
                options: {
                    spades: {
                        particles: {
                            color: {
                                value: "#000000",
                            },
                        },
                    },
                    hearts: {
                        particles: {
                            color: {
                                value: "#ff0000",
                            },
                        },
                    },
                    diamonds: {
                        particles: {
                            color: {
                                value: "#ff0000",
                            },
                        },
                    },
                    clubs: {
                        particles: {
                            color: {
                                value: "#000000",
                            },
                        },
                    },
                },
            },
            opacity: {
                value: 1,
            },
            size: {
                value: 30,
            },
            move: {
                enable: true,
                speed: 2,
            },
        },
        interactivity: {
            events: {
                onHover: {
                    enable: true,
                    mode: "repulse",
                },
                onClick: {
                    enable: true,
                    mode: "push",
                },
            },
            modes: {
                repulse: {
                    distance: 200,
                },
                push: {
                    quantity: 4,
                },
            },
        },
        background: {
            color: "#fff",
        },
    };
    exports.default = options;
});
