// tsparticles-demo-configs/umd/c/colorAnimation.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Color Animation",
        particles: {
            number: {
                value: 80,
                density: {
                    enable: true,
                },
            },
            color: {
                value: "#ff0000",
                animation: {
                    count: 1,
                    enable: true,
                    speed: 60,
                    sync: true,
                },
            },
            stroke: {
                width: 30,
                color: {
                    value: "#0000ff",
                    animation: {
                        count: 1,
                        enable: true,
                        speed: 60,
                        sync: true,
                    },
                },
            },
            shape: {
                type: "circle",
            },
            opacity: {
                value: 1,
            },
            size: {
                value: 15,
            },
            move: {
                enable: true,
                speed: 6,
            },
        },
        background: {
            color: "#000000",
        },
    };
    exports.default = options;
});
