// tsparticles-demo-configs/umd/c/index.js
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "./cards", "./chars", "./clickPause", "./collisionsAbsorb", "./collisionsBounce", "./collisionsDestroy", "./colorAnimation", "./connect", "./customPreset", "./customShape"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const cards_1 = __importDefault(require("./cards"));
    const chars_1 = __importDefault(require("./chars"));
    const clickPause_1 = __importDefault(require("./clickPause"));
    const collisionsAbsorb_1 = __importDefault(require("./collisionsAbsorb"));
    const collisionsBounce_1 = __importDefault(require("./collisionsBounce"));
    const collisionsDestroy_1 = __importDefault(require("./collisionsDestroy"));
    const colorAnimation_1 = __importDefault(require("./colorAnimation"));
    const connect_1 = __importDefault(require("./connect"));
    const customPreset_1 = __importDefault(require("./customPreset"));
    const customShape_1 = __importDefault(require("./customShape"));
    exports.default = {
        cards: cards_1.default,
        chars: chars_1.default,
        clickPause: clickPause_1.default,
        collisionsAbsorb: collisionsAbsorb_1.default,
        collisionsBounce: collisionsBounce_1.default,
        collisionsDestroy: collisionsDestroy_1.default,
        colorAnimation: colorAnimation_1.default,
        connect: connect_1.default,
        customPreset: customPreset_1.default,
        customShape: customShape_1.default,
    };
});
