// tsparticles-demo-configs/umd/i/infection.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Infection",
        infection: {
            enable: true,
            infections: 10,
            cure: true,
            stages: [
                {
                    color: "#ff0000",
                    duration: 1,
                },
                {
                    color: "#ffa500",
                    duration: 1,
                    rate: 2,
                },
                {
                    color: "#ffff00",
                    duration: 1,
                    rate: 2,
                },
                {
                    color: "#008000",
                    duration: 1,
                    rate: 3,
                },
                {
                    color: "#0000ff",
                    duration: 1,
                    rate: 4,
                },
                {
                    color: "#4b0082",
                    duration: 1,
                    rate: 5,
                },
                {
                    color: "#ee82ee",
                    duration: 1,
                    rate: 6,
                    infectedStage: 0,
                },
            ],
        },
        particles: {
            collisions: {
                enable: true,
            },
            number: {
                value: 400,
                density: {
                    enable: true,
                },
            },
            color: {
                value: "#ffffff",
            },
            shape: {
                type: "circle",
            },
            opacity: {
                value: 0.8,
            },
            size: {
                value: 5,
            },
            move: {
                enable: true,
                speed: 20,
                outModes: "bounce",
            },
        },
        background: {
            color: "#000000",
        },
    };
    exports.default = options;
});
