// tsparticles-demo-configs/umd/b/big.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Big Particles",
        particles: {
            number: {
                value: 30,
            },
            color: {
                value: ["#5bc0eb", "#fde74c", "#9bc53d", "#e55934", "#fa7921"],
            },
            shape: {
                type: "circle",
            },
            opacity: {
                value: {
                    min: 0.4,
                    max: 0.8,
                },
            },
            size: {
                value: {
                    min: 300,
                    max: 400,
                },
                animation: {
                    enable: true,
                    speed: 100,
                    sync: false,
                },
            },
            move: {
                enable: true,
                speed: 10,
                direction: "top",
            },
        },
        background: {
            color: "#ffffff",
        },
    };
    exports.default = options;
});
