// ng-fireworks/esm2022/lib/ng-fireworks.module.mjs
import { NgModule } from '@angular/core';
import { NgFireworksComponent } from './ng-fireworks.component';
import * as i0 from "@angular/core";
export class NgFireworksModule {
    static { this.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule }); }
    static { this.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule, declarations: [NgFireworksComponent], exports: [NgFireworksComponent] }); }
    static { this.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule }); }
}
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [NgFireworksComponent],
                    imports: [],
                    exports: [NgFireworksComponent],
                }]
        }] });
