// ng-fireworks/esm2022/lib/ng-fireworks.component.mjs
import { isPlatformServer } from '@angular/common';
import { Component, Inject, Input, PLATFORM_ID } from '@angular/core';
import { fireworks } from 'tsparticles-fireworks';
import * as i0 from "@angular/core";
export class NgFireworksComponent {
    constructor(platformId) {
        this.platformId = platformId;
        this.id = 'tsparticles';
    }
    ngAfterViewInit() {
        if (isPlatformServer(this.platformId)) {
            return;
        }
        fireworks(this.id, this.options);
    }
    static { this.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksComponent, deps: [{ token: PLATFORM_ID }], target: i0.ɵɵFactoryTarget.Component }); }
    static { this.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "14.0.0", version: "16.1.8", type: NgFireworksComponent, selector: "ng-fireworks", inputs: { options: "options", id: "id" }, ngImport: i0, template: ` <div [id]="id"></div>`, isInline: true }); }
}
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksComponent, decorators: [{
            type: Component,
            args: [{ selector: 'ng-fireworks', template: ` <div [id]="id"></div>` }]
        }], ctorParameters: function () { return [{ type: undefined, decorators: [{
                    type: Inject,
                    args: [PLATFORM_ID]
                }] }]; }, propDecorators: { options: [{
                type: Input
            }], id: [{
                type: Input
            }] } });
