// ng-fireworks/fesm2022/ng-fireworks.mjs
import { isPlatformServer } from '@angular/common';
import * as i0 from '@angular/core';
import { PLATFORM_ID, Component, Inject, Input, NgModule } from '@angular/core';
import { fireworks } from 'tsparticles-fireworks';

class NgFireworksComponent {
    constructor(platformId) {
        this.platformId = platformId;
        this.id = 'tsparticles';
    }
    ngAfterViewInit() {
        if (isPlatformServer(this.platformId)) {
            return;
        }
        fireworks(this.id, this.options);
    }
    static { this.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksComponent, deps: [{ token: PLATFORM_ID }], target: i0.ɵɵFactoryTarget.Component }); }
    static { this.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "14.0.0", version: "16.1.8", type: NgFireworksComponent, selector: "ng-fireworks", inputs: { options: "options", id: "id" }, ngImport: i0, template: ` <div [id]="id"></div>`, isInline: true }); }
}
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksComponent, decorators: [{
            type: Component,
            args: [{ selector: 'ng-fireworks', template: ` <div [id]="id"></div>` }]
        }], ctorParameters: function () { return [{ type: undefined, decorators: [{
                    type: Inject,
                    args: [PLATFORM_ID]
                }] }]; }, propDecorators: { options: [{
                type: Input
            }], id: [{
                type: Input
            }] } });

class NgFireworksModule {
    static { this.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule }); }
    static { this.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule, declarations: [NgFireworksComponent], exports: [NgFireworksComponent] }); }
    static { this.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule }); }
}
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgFireworksModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [NgFireworksComponent],
                    imports: [],
                    exports: [NgFireworksComponent],
                }]
        }] });

/*
 * Public API Surface of ng-fireworks
 */

/**
 * Generated bundle index. Do not edit.
 */

export { NgFireworksComponent, NgFireworksModule };
