// ng-confetti/esm2022/lib/ng-confetti.module.mjs
import { NgModule } from '@angular/core';
import { NgConfettiComponent } from './ng-confetti.component';
import * as i0 from "@angular/core";
export class NgConfettiModule {
    static { this.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgConfettiModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule }); }
    static { this.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "16.1.8", ngImport: i0, type: NgConfettiModule, declarations: [NgConfettiComponent], exports: [NgConfettiComponent] }); }
    static { this.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgConfettiModule }); }
}
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "16.1.8", ngImport: i0, type: NgConfettiModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [NgConfettiComponent],
                    imports: [],
                    exports: [NgConfettiComponent],
                }]
        }] });
