// tsparticles-demo-configs/umd/t/index.js
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "./test", "./textMask", "./textMaskMultiline", "./themes", "./tilt", "./trail", "./trailImage", "./triangles", "./twinkle"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const test_1 = __importDefault(require("./test"));
    const textMask_1 = __importDefault(require("./textMask"));
    const textMaskMultiline_1 = __importDefault(require("./textMaskMultiline"));
    const themes_1 = __importDefault(require("./themes"));
    const tilt_1 = __importDefault(require("./tilt"));
    const trail_1 = __importDefault(require("./trail"));
    const trailImage_1 = __importDefault(require("./trailImage"));
    const triangles_1 = __importDefault(require("./triangles"));
    const twinkle_1 = __importDefault(require("./twinkle"));
    exports.default = {
        test: test_1.default,
        textMask: textMask_1.default,
        textMaskMultiline: textMaskMultiline_1.default,
        themes: themes_1.default,
        tilt: tilt_1.default,
        trail: trail_1.default,
        trailImage: trailImage_1.default,
        triangles: triangles_1.default,
        twinkle: twinkle_1.default,
    };
});
