// tsparticles-demo-configs/umd/t/test.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Test",
        fullScreen: {
            enable: false,
        },
        background: {
            color: "#000",
        },
        particles: {
            color: {
                value: "#ffffff",
            },
            size: {
                value: 6,
            },
            links: {
                enable: true,
                distance: 200,
            },
            number: {
                value: 80,
                density: {
                    enable: true,
                },
            },
            opacity: {
                value: 0.5,
            },
            shape: {
                type: "circle",
            },
        },
    };
    exports.default = options;
});
