// tsparticles-demo-configs/umd/p/planes.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Planes",
        interactivity: {
            events: {
                onClick: {
                    enable: true,
                    mode: "push",
                },
            },
            modes: {
                push: {
                    quantity: 4,
                },
            },
        },
        particles: {
            color: {
                value: "#ffffff",
            },
            move: {
                enable: true,
                speed: 6,
            },
            number: {
                density: {
                    enable: true,
                },
                value: 80,
            },
            rotate: {
                value: 45,
                path: true,
            },
            opacity: {
                value: 1,
            },
            shape: {
                options: {
                    image: {
                        height: 128,
                        src: "https://particles.js.org/images/plane_alt.png",
                        width: 128,
                    },
                },
                type: "image",
            },
            size: {
                value: {
                    min: 16,
                    max: 32,
                },
            },
        },
        detectRetina: true,
    };
    exports.default = options;
});
