// tsparticles-demo-configs/umd/e/emitterAngled.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const options = {
        name: "Emitter Angled",
        particles: {
            number: {
                value: 0,
            },
            color: {
                value: "random",
            },
            shape: {
                type: "circle",
            },
            opacity: {
                value: 0.3,
            },
            size: {
                value: {
                    min: 5,
                    max: 10,
                },
            },
            move: {
                angle: {
                    offset: 0,
                    value: 30,
                },
                enable: true,
                speed: 15,
                direction: "top",
                outModes: {
                    default: "destroy",
                },
            },
        },
        background: {
            color: "#fff",
        },
        emitters: [
            {
                direction: "top",
                position: {
                    x: {
                        min: 25,
                        max: 75,
                    },
                    y: 100,
                },
                life: {
                    duration: 3,
                    delay: 5,
                    count: 0,
                },
            },
        ],
    };
    exports.default = options;
});
