// tsparticles-demo-configs/umd/e/index.js
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "./emitter", "./emitterAbsorber", "./emitterAngled", "./emitterImages", "./emitterPaths", "./emitterShapes"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    const emitter_1 = __importDefault(require("./emitter"));
    const emitterAbsorber_1 = __importDefault(require("./emitterAbsorber"));
    const emitterAngled_1 = __importDefault(require("./emitterAngled"));
    const emitterImages_1 = __importDefault(require("./emitterImages"));
    const emitterPaths_1 = __importDefault(require("./emitterPaths"));
    const emitterShapes_1 = __importDefault(require("./emitterShapes"));
    exports.default = {
        emitter: emitter_1.default,
        emitterAbsorber: emitterAbsorber_1.default,
        emitterAngled: emitterAngled_1.default,
        emitterImages: emitterImages_1.default,
        emitterPaths: emitterPaths_1.default,
        emitterShapes: emitterShapes_1.default,
    };
});
